src/DataFixtures/UserFixtures.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\Technology;
+use App\Entity\User;
+use DateTime;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+use Faker\Factory;
+
+class UserFixtures extends Fixture
+{
+    public const ROLES = [
+        'dev',
+        'po',
+        'qa'
+    ];
+
+    public function load(ObjectManager $manager): void
+    {
+        $faker = Factory::create();
+
+        $user = new User();
+        $user->setUsername('ADMIN');
+        $user->setPassword('iamadmin');
+        $user->setRole('undefined');
```

Comment:
`role` will be later user to define the access level. Rename the field to `position`. Also don't use undefined .. use one from the `self::ROLES`